### DISCUSSION TITLE
Auto-type for a specific website

### DISCUSSION THREAD

**Discussion by rpedrica** [keepassxreboot/keepassxc#11238] [Q&A] [answered]
I'm having trouble getting auto-type to work on a specific site - system details:

Firefox 128 with keepassxc plugin 1.9.3
KeepassXC 2.7.9
Linux Slackware -current

The site is:

https://timesheets.secrecruitment.com/login/auth

The site implements a 2 page/step login process:

1. username on pg.1
2. password on pg.2

I've created an auto-type sequence that directly matches the manual steps one uses to login (tested):

{CLEARFIELD}{USERNAME}{ENTER}{DELAY 5}{CLEARFIELD}{PASSWORD}{TAB}{ENTER}

The first part ({CLEARFIELD}{USERNAME}{ENTER}) inserts the username and enters to move onto the next page.
The 2nd part ({DELAY 5}{CLEARFIELD}{PASSWORD}{TAB}{ENTER}) inserts the password and enters to complete the login.

On performing the auto-type sequence, the browser (Firefox) shows the username being entered but then the password appears to be inserted as a search query in the browser itself (ie. ctrl-f to search for something in the page) and the rest of the login fails.

In Chrome, the behaviour is slightly different (but still fails) - the username is inserted and you are moved onto the 2nd page, but the full password does not appear to be inserted into the password field (looking at the amount of wildcarded chars listed). I assume only a few chars from the password are being inserted looking at this visually.

I've done a screen capture which I hope shows the issue.

https://youtu.be/2hZTzfL_Dmg

Looking for ideas as it's painful to insert the login creds manually. Help appreciated.

Regards, Robby

**Comment by droidmonkey** [comment, marked as answer]
Delay is measured in _milliseconds_ so you want {DELAY 5000}

https://keepassxc.org/docs/KeePassXC_UserGuide#_configure_auto_type_sequences

**Comment by rpedrica** [comment]
@droidmonkey I can feel Mr. T behind me whispering "pity the fool"! I don't know how I got that one so wrong! Thanks and it's working now.
